packages/mobile/src/fiatExchanges/ExternalExchanges.tsx

```diff
@@ -68,12 +68,10 @@ function ExternalExchanges({ route }: Props) {
             currency: route.params.currency === CURRENCY_ENUM.DOLLAR ? t('celoDollars') : 'CELO',
           })}
         </Text>
-        <View style={styles.accountNumberContainer}>
-          <View style={styles.accountNoTextContainer}>
-            <Text style={styles.accountNoText}>Account</Text>
-            <Text style={styles.accountNoText}>No.</Text>
-          </View>
+        <View style={styles.accountBox}>
+          <Text style={styles.accountLabel}>{t('sendFlow7:accountNumberLabel')}</Text>
           <AccountNumber address={account || ''} location={Screens.ExternalExchanges} />
+          <Text style={styles.link}>{t('accountScreen10:tapToCopy')}</Text>
```

Comment:
Nothing to do in this PR, but it seems we could refactor this since they (`AccountNumber` and `Tap to copy`) are often used together.
Either extending `AccountNumber` (for instance with an optional `showTap` prop) or creating a new component wrapping the 2.